en/04_Changelogs/beta/5.0.0-beta1.md

```diff
@@ -11,7 +66,86 @@ We will only consider merging post-beta breaking API changes if they are necessa
 
 Non-breaking API changes and enhancements may be merged after the initial CMS 5 beta release.
 
-## Post CMS5 RC freeze
+#### Post CMS5 RC freeze
 Once the initial CMS5.0 RC is released, non-breaking API changes will no longer be accepted and will have to be targeted to CMS5.1.
 
 Bug fixes may be accepted if they can be solved without changing APIs.
+
+### Known issues
+
+The Silverstripe CMS 5 beta 1 release contains several know issues. We'll aim to address those prior the Silverstripe CMS 5 Release candidate. These likely only the tip of the iceberg.
+
+#### Bugs
+
+- [Yubikey registration fails](https://github.com/silverstripe/silverstripe-webauthn-authenticator/issues/124)
+- [Fix broken JavaScript tests](https://github.com/silverstripe/silverstripe-admin/issues/1421)
+- [Clear up errors and warnings when running yarn commands](https://github.com/silverstripe/silverstripe-admin/issues/1416)
+- [React context no longer has "You have unsaved changes" confirm dialog](https://github.com/silverstripe/silverstripe-admin/issues/1409)
+- [Cannot delete image in WYSIWYG with delete button](https://github.com/silverstripe/silverstripe-admin/issues/1410)
+- [Console error whenever a toast pops up](https://github.com/silverstripe/silverstripe-admin/issues/1412)
+
+#### Partial build failures
+
+We have partial build failures on these modules.
```

Comment:
What does "partial build failure" even mean?
Do we just mean "at least one CI job is failing"? If so, lets say "CI build failures" as the heading and then explain exactly what the situation is in the description. Something like "Some of our modules are failing one or more Continuous Integration job. We've looked at these and decided that none of the failures is critical or warrants holding back the beta release. These are all either problems with out CI build system itself or indications of non-critical regressions."

Reply:
I still your explanatory snippet.